dev/githubanalysis/lib/main.dart

```diff
@@ -61,6 +61,11 @@ bool issueIsClosed(final FullIssue issue) {
   return issue.metadata.isClosed || issue.metadata.state.toUpperCase() == 'CLOSED';
 }
 
+// Turns a username into an internal canonicalized form.
+// We add a "👤" emoji here so that if we accidentally use the canonicalized form
+// in the output, we will catch it.
```

Comment:
Where do we check for this?

Reply:
I just meant we'll notice it in the console.